src/System.Reflection/tests/System.Reflection.Tests.csproj

```diff
@@ -25,6 +27,9 @@
     <Compile Include="TypeInfoTests.cs" />
     <Compile Include="ExceptionTests.cs" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)' == 'netstandard'">
+    <Compile Include="FieldInfoTests.netstandard.cs" />
```

Comment:
I like using the version-less ids here. When we remap this to a a higher version in the future it doesn't invalidate this condition. The only think that would invalidate it is if we down versioned it but we don't plan to do that.